test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
- [ ] Kindly ensure all your test files have test in them , its not good practise to have empty test file.

Reply:
## Kindly check well I use the correct gitflow, I create the dev branch and I make  it  the default branch and create the feature branches and I submit the pull request for my dev to master after merging all branches to dev please check very well the gitflow documentation

Reply:
## Please check all test in spec folder